index.html

```diff
@@ -1,232 +1,326 @@
-                    </div>
-                    <div class="frame">
-                        <img src="/images/html.png" alt="">
-                        <span>html</span>
-                    </div>
-                    <div class="frame">
-                        <img src="/images/css.png" alt="">
-                        <span>css</span>
-                    </div>  
-                </div>                
-                <div class="frame8">
-                    <p class="lang-item">Frameworks<span><img src="/images/Disabled.png" alt=""></span></p>  
-                </div>
-                <div class="frame8">
-                    <p class="lang-item">Skills<span><img src="/images/Disabled1.png" alt=""></span></p>  
-                </div>
-            </div>
-        </section>
-        <footer>
-            <section id="contact-form">
-                <h4>Contact me</h4>
-                <p>
-                If you have an application you are interested in developing,
-                a feature that you need built or a project that needs coding.
-                I’d love to help with it
-                </p>
-                <form class="input" action="https://formspree.io/f/mayvgkrd" method="POST">
-                   <input type="text" class="input-item" name="Name" placeholder="full name" maxlength="30" required>
-                    <input type="email" id="email" class="input-item" name="Email" placeholder="email" required>
-                   <textarea name="message" id="msg" cols="30" rows="30" placeholder="write your message here" maxlength="500" required></textarea>
-                   <small class="error"></small>
-                   <button type="submit" id="submit">Get in Touch</button>
-                </form>                                                                                             
-            </section>
-         </footer>
-    </main>
-</body>
-</html>
+      </section>
+    </section>
+    <section class="about-me">
+      <div class="aboutme2">
+        <h1 class="about-my-self">
+          About <br />
+          Myself
+        </h1>
+        <p class="contact-me">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h3 class="letsconnect">LET'S CONNECT</h3>
+        <ul class="socialmedia-list">
+          <li>
+            <img src="images/Vector.png" alt="v1" />
+          </li>
+          <li>
+            <img src="images/Linkedinicon.png" alt="linkv" />
+          </li>
+
+          <li><img src="images/med.png" alt="medv" /></li>
+          <li><img src="images/cat.png" alt="catv" /></li>
+          <li><img src="images/angelist.png" alt="angv" /></li>
+        </ul>
+        <button type="button" class="btn">Get my resume</button>
```

Comment:
- [ ] I really appreciate your efforts but I think it would be better if you add some margin here or padding to match the design as much as possible 💪🏻💪🏻💯💯

- Your design 
![image](https://user-images.githubusercontent.com/58553711/179331242-f0f37355-4663-44e7-bf5b-f50d9c08cad6.png)

- figma design
![image](https://user-images.githubusercontent.com/58553711/179331354-993092b8-3d5b-42ba-8ee0-94a063035dbf.png)